about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>aboutPage</title>
+    <script
+      src="https://kit.fontawesome.com/4339e938ba.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <link rel="stylesheet" href="style.css" />
+  <body>
+    <div class="menu-bar hidden-menu">
+      <button class="close-btn">
+              <i class="fa fa-times"></i>
+      </button>
+      <nav class="menu-mobile">
+           <a href="index.html">Home</a>
+            <a href="#contact">Contact</a>
+            <a href="#live">Live</a>
+      </na>
+    </div>
+    <header class="about-header">
+      <a class="hum-burger"><i class="fa-solid fa-bars"></i></a>
+    </header>
+
+ <section class="desktop-nav">
+
+          <div class="social-links">
+            <a href="#"><i class="fa-brands fa-facebook"></i></a></a>
+          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          <a href="#">Logout</a>
+          <a href="#">myPage</a>
+          <a href="#">Korean</a>
+        </div>
+        <div class="logo-container">
+          <a href="index.html"><img src="images/bg-visit.jpg" alt="logo" class="logo"></a>
+          
+          <ul class="info-lists">
+              <li><a href="about.html" class="red-about">About</a></li>
+             <li><a href="">Program</a></li>
+             <li><a href="">Join</a></li>
+              <li><a href="">Sponsor</a></li>
+               <li><a href="">News</a></li>
+                <li class="red-border"><a href="">CC Campaign</a></li>
+            </ul>
+        </div>
+ </section>
+
+
+    <section class="home-page">
```

Comment:
- [ ] As stated by the previous reviewers, kindly centralize the contents of this section to match the given template.
### Your design
![image](https://user-images.githubusercontent.com/94551484/217915995-b6cb0a10-c0f6-4818-8850-4669871375c7.png)
### Design template
![f1](https://user-images.githubusercontent.com/94551484/217916281-c7ea492d-40c3-4728-ad50-c458ce908b10.png)